container_registry/container_analysis/src/sample/samples.go

```diff
@@ -135,29 +149,44 @@ func getNote(ctx context.Context, client *containeranalysis.GrafeasV1Beta1Client
 	return note, err
 }
 
-// [END get_note]
+// [END containeranalysis_get_note]
 
-// [START get_occurrence]
+// [START containeranalysis_get_occurrence]
 
 // getOccurrence retrieves and prints a specified Occurrence from the server.
-// occurrenceName should be in the following format: "projects/[PROJECT_ID]/occurrences/[OCCURRENCE_ID]"
-func getOccurrence(ctx context.Context, client *containeranalysis.GrafeasV1Beta1Client, occurrenceName string) (*grafeaspb.Occurrence, error) {
-	req := &grafeaspb.GetOccurrenceRequest{Name: occurrenceName}
+func getOccurrence(occurrenceID, projectID string) (*grafeaspb.Occurrence, error) {
+	ctx := context.Background()
+	client, err := containeranalysis.NewGrafeasV1Beta1Client(ctx)
+	if err != nil {
+		return nil, err
+	}
+	defer client.Close()
+
+	req := &grafeaspb.GetOccurrenceRequest{
+		Name: fmt.Sprintf("projects/%s/occurrences/%s", projectID, occurrenceID),
+	}
 	occ, err := client.GetOccurrence(ctx, req)
 	fmt.Println(occ)
```

Comment:
Was already here, but would you mind updating to print to an `io.Writer`? This gets printed while running tests and makes it more difficult to diagnose issues.

https://github.com/GoogleCloudPlatform/golang-samples/blob/master/CONTRIBUTING.md#print-to-an-iowriter